web/packages/teleport/src/Discover/ConnectMyComputer/SetupConnect/SetupConnect.tsx

```diff
@@ -80,15 +186,87 @@ export function SetupConnect(props: AgentStepProps) {
         </ButtonSecondary>
       </StyledBox>
 
+      {pollingStatus}
+
       <ActionButtons
-        onProceed={() => {}}
-        disableProceed={true}
+        onProceed={props.nextStep}
+        disableProceed={!node}
         onPrev={props.prevStep}
       />
     </Flex>
   );
 }
 
+/**
+ * usePollForConnectMyComputerNode polls for a Connect My Computer node that joined the cluster
+ * after starting opening the SetupConnect step.
+ *
+ * The first polling request fills out a set of node IDs (initialNodeIdsRef). Subsequent requests
+ * check the returned nodes against this set. The hook stops polling as soon as a node that is not
+ * in the set was found.
```

Comment:
i think we should add/clarify that there can be multiple nodes with label `teleport.dev/connect-my-computer/owner: <current-username>` which is why we require the (initialNodeIdsRef) and we want the one added latest?